src/js/base-styles.js

```diff
@@ -6,13 +6,17 @@
 import window from 'global/window';
 import document from 'global/document';
 
-if (window.VIDEOJS_NO_BASE_THEME) return;
+if (window.VIDEOJS_NO_BASE_THEME) {
+  return;
+}
 
 const styles = '{{GENERATED_STYLES}}';
 
-if (styles === '{{GENERATED'+'_STYLES}}');
+// Don't think we need this as it's a noop?
+// if (styles === '{{GENERATED'+'_STYLES}}');
```

Comment:
`base-styles` isn't even used anywhere anyway right now, so, feel free to just leave as is.